Source/Blazorise/Components/Rating/Rating.razor.cs

```diff
@@ -225,5 +226,13 @@ public int SelectedValue
     /// </summary>
     [Parameter] public EventCallback<int?> HoveredValueChanged { get; set; }
 
+    /// <summary>
+    /// Expression that provides the tooltip for the rating item to have additional user-friendly information.
+    /// </summary>
+    /// <remarks>
+    /// The expression receives the rating item <Code>Value</Code> to determine the result.
+    /// </remarks>
+    [Parameter] public Func<int?, RatingTooltip> GetTooltip { get; set; }
```

Comment:
Is there a Rating Item that's represented by null? That's why we have nullable int? Would we have an actual tooltip render when the value is null? What happens?

Reply:
Made*? :P